triggers/test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,70 @@
+Manifest-Version: 1.0
```

Comment:
What does this file do? I've never seen these before

Reply:
Some build artifact? Prob can be removed

Reply:
I removed it - it was generated by the daml compiler but I'm not sure why...